src/output/icons.rs

```diff
@@ -74,6 +74,7 @@ impl Icons {
     const LANG_RUBYRAILS: char  = '\u{e73b}';  // 
     const LANG_RUST: char       = '\u{e68b}';  // 
     const LANG_SASS: char       = '\u{e603}';  // 
+    const LANG_SCHEME: char     = '\u{e6b1}';  // 
```

Comment:
The `\u{e6b1}` I believe is correct but the comment you have 
![image](https://github.com/eza-community/eza/assets/121899304/cf3227e9-6d57-42cf-96ea-a9c17c05b3f7)
seems to be the erlang logo

Reply:
Weird. This is what it looks like for me. I suppose this part of Unicode is the "Private Use Area" and a free-for-all. For what it is worth I am using the Nerd Fonts version of GitHub's Monaspace. I could change for `\u{03bb}`  (GREEK SMALL LETTER LAMDA) but I figured there was a reason the other icons were not in that range.
![image](https://github.com/eza-community/eza/assets/93013864/28abff56-e3c5-4998-ab89-681d4abbf83f)

Reply:
That is odd because I actually use the same font :rofl: